modules/platforms/dotnet/Apache.Ignite.Core.Tests/Client/ClientSecurityPermissionsTest.cs

```diff
@@ -86,7 +86,7 @@ public void TestDataStreamerNoPermissionThrowsSecurityViolationClientException([
                 var clientEx = (IgniteClientException)ex.GetBaseException();
 
                 Assert.AreEqual(ClientStatusCode.SecurityViolation, clientEx.StatusCode);
-                Assert.AreEqual("Client is not authorized to perform this operation", clientEx.Message);
+                Assert.IsTrue(clientEx.Message.StartsWith("Client is not authorized to perform this operation"));
```

Comment:
We should not include the stack trace by default, error messages should not change as a result of this PR.

Reply:
fixed